README.md

```diff
@@ -1,57 +1,73 @@
-Portfolio
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-    My portfolio in 2021
+# Project Name
+  Portfolio
+> My portfolio in 2021. To be updated.
 
-image
+![screenshot](https://github.com/Somdotta07/Portfolio/blob/Feature/media/screenshot.PNG)
 
+It's a portfolio project and template taken from Figma
 
-The portfolio is made of HTML and CSS . To be updated.
-Built With
+## Built With
 
-    HTML and CSS
+- HTML and CSS
+- NA
+- NA
 
-Live Demo
-http://127.0.0.1:5500/index.html
+## Live Demo
 
-Getting Started:
+[Live Demo Link]( http://127.0.0.1:5500/index.html)
 
 
-Prerequisites: VS code editor and gitbash
-Setup :
+## Getting Started
 
-    In the VS code add 1 HTML and 1 CSS file
-    add a copy of [.github/workflows/linters.yml] to .github/workflows folder
-    Add .gitignore file where we are keeping the files which we have to ignore.
-    Add linter files .hintrc and .stylelintrc.json to fix the errors
-    Take template from Figma
-    Add icons with <a> tag
-   
+### Prerequisites
+VS code editor and gitbash 
 
-Install: node.js
-Usage:
+### Setup
+In the VS code add 1 HTML and 1 CSS file
+add a copy of [.github/workflows/linters.yml] to .github/workflows folder
+Add .gitignore file where we are keeping the files which we have to ignore.
+Add linter files .hintrc and .stylelintrc.json to fix the errors
+Take template from Figma
+Add icons with <a> tag
+Add images & buttons
+Use Flexbox & Grid for styling
 
+### Install
+   node.js
+
+### Usage
 To open the project you can simply download and double click the index.html file
-    
-Authors
+
+
+
+
+## Authors
 
 👤 Somdotta Sarkar
 
-    GitHub: https://github.com/Somdotta07
-    Twitter:https://twitter.com/iloveenv
-    LinkedIn: https://www.linkedin.com/in/somdotta-sarkar-8849b419/
+- GitHub: https://github.com/Somdotta07
+- Twitter:https://twitter.com/iloveenv
+- LinkedIn: https://www.linkedin.com/in/somdotta-sarkar-8849b419/
```

Comment:
- [ ] **[OPTIONAL]** This is only a suggestion. It would be nice to format your links like this:
```
- [Github](https://github.com/Somdotta07)
- [Twitter](https://github.com/Somdotta07)
- [LinkedIn](https://www.linkedin.com/in/somdotta-sarkar-8849b419/)
```

Reply:
Thanks